ocw_data_parser/ocw_data_parser.py

```diff
@@ -261,14 +264,17 @@ def extract_media_locally(self):
         path_to_containing_folder = self.destination_dir + "static_files/"
         os.makedirs(path_to_containing_folder, exist_ok=True)
         for j in self.media_jsons:
-            filename = safe_get(j, "_uid") + "_" + safe_get(j, "id")
+            file_name = safe_get(j, "_uid") + "_" + safe_get(j, "id")
             d = get_binary_data(j)
             if d:
-                with open(path_to_containing_folder + filename, "wb") as f:
+                with open(path_to_containing_folder + file_name, "wb") as f:
                     data = base64.b64decode(d)
                     f.write(data)
-                update_file_location(self.master_json, path_to_containing_folder + filename, safe_get(j, "_uid"))
-                log.info("Extracted %s", filename)
+                if self.static_prefix:
```

Comment:
if `self.static_prefix` is set to anything besides "static_files/", the media will not load in an HTML page.  I wonder if `self.static_prefix` should be set by default to "static_files/", and then change `path_containing_folder` to
```python
path_to_containing_folder = self.destination_dir + self.static_prefix
```

if there's still a need to sometimes use the original `path_to_containing_folder` instead of `self.static_prefix` as the string passed to `update_file_location`, maybe you could introduce a boolean kwarg like `use_static_prefix`?

```python
url_path_to_media = self.static_prefix if use_static_prefix else path_to_containing_folder
...
update_file_location(self.master_json, url_path_to_media + file_name, ...etc
```

What do you think?

Reply:
So I had actually been thinking about this.  I think you're right in that path_to_containing_folder should append the static prefix in the output directory.  I can't think of a situation in which you would want the URL passed into update_file_location to contain the destination directory, aside from when a static prefix is not passed in.  I think maybe a boolean flag is a bit unnecessary, as I would assume the fact that it has been passed in means it should be used.  Something like this:

`url_path_to_media = self.static_prefix if self.static_prefix else path_to_containing_folder`

As long as we set the file write location to match what is passed in, I think that's enough?